onc/Onc.m

```diff
@@ -128,7 +141,9 @@
             this.tree = temp.tree;
             %These codes can then be used for input to onc.getDevices by
             %providing the locationCodes
-            
+
+            % check if it's running the latest version. If not, throw a warning
+            util.checkVersion;
```

Comment:
It's more clear that you are calling a function if you call it like util.checkVersion()